lib/filewatch/read_mode/handlers/read_zip_file.rb

```diff
@@ -49,8 +49,8 @@ def handle_specifically(watched_file)
     def close_and_ignore_ioexception(closeable)
       begin
         closeable.close
-      rescue Exception # IOException can be thrown by any of the Java classes that implement the Closable interface.
-        logger.warn("Ignoring an IOException when closing an instance of #{closeable.class.name}")
+      rescue Exception => e # IOException can be thrown by any of the Java classes that implement the Closable interface.
+        logger.warn("Ignoring an IOException when closing an instance of #{closeable.class.name}", "exception" => e.class.name, "message" => e.message)
```

Comment:
I believe our "normal" exception logging pattern is more like this no?
```
logger.warn("some problem happened", :exception => e, :backtrace => e.backtrace)
```

Reply:
I am a little bit wary of passing an object as the value in a logging statement because it simply calls `to_s` on the object.
I will change this case to `:exception => e`

We don't need the backtrace in this case because we know exactly where it is happening.

Reply:
👍